api/openapi/v2/core-metadata.yaml

```diff
@@ -695,415 +715,142 @@ components:
-      responses:
-        '204':
-          description: "Delete successful"
-          headers:
-            X-Correlation-ID:
-              $ref: '#/components/headers/correlatedResponseHeader'
-        '400':
-          description: "Request is in an invalid state"
-          headers:
-            X-Correlation-ID:
-              $ref: '#/components/headers/correlatedResponseHeader'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/ErrorResponse'
-        '500':
-          description: "An unexpected error occurred on the server"
-          headers:
-            X-Correlation-ID:
-              $ref: '#/components/headers/correlatedResponseHeader'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/ErrorResponse'
-  /addressable/port/{port}:
-    parameters:
-      - $ref: '#/components/parameters/correlatedRequestHeader'
-      - name: port
-        in: path
-        required: true
-        schema:
-          type: integer
-        description: "The port of the addressable you wish to load."
-      - $ref: '#/components/parameters/offsetParam'
-      - $ref: '#/components/parameters/limitParam'
-    get:
-      summary: "Returns a list of all addressables listening on the specified port"
-      responses:
-        '200':
-          description: "OK"
-          headers:
-            X-Correlation-ID:
-              $ref: '#/components/headers/correlatedResponseHeader'
-          content:
-            application/json:
-              schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/AddressableResponse'
-        '404':
-          description: "The requested resource does not exist"
-          headers:
-            X-Correlation-ID:
-              $ref: '#/components/headers/correlatedResponseHeader'
-          content:
-            application/json:
-              schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/ErrorResponse'
-        '500':
-          description: "An unexpected error occurred on the server"
-          headers:
-            X-Correlation-ID:
-              $ref: '#/components/headers/correlatedResponseHeader'
-          content:
-            application/json:
-              schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/ErrorResponse'
+      required:
+        - id
+        - deviceServiceName
+    UpdateDeviceServiceResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      description: "A response type for use in updating a device service. The ID of the affected record will be returned to the caller."
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+      required:
+        - id
+    VersionRequest:
```

Comment:
VersionRequest goes away as we agreed that this witl be a GET request rather than a PUT.

Reply:
Removed versionRequest.